src/xrGame/visual_memory_manager.cpp

```diff
@@ -313,14 +318,20 @@ float CVisualMemoryManager::get_object_velocity(
     return (pos1.vPosition.distance_to(pos0.vPosition) / (float(pos1.dwTime) / 1000.f - float(pos0.dwTime) / 1000.f));
 }
 
-float CVisualMemoryManager::get_visible_value(
+float CVisualMemoryManager::get_visible_value(const CGameObject* game_object,
     float distance, float object_distance, float time_delta, float object_velocity, float luminocity) const
 {
     float always_visible_distance = current_state().m_always_visible_distance;
 
     if (distance <= always_visible_distance + EPS_L)
         return (current_state().m_visibility_threshold);
 
+    //Alundaio: hijack not_yet_visible_object to lua
+    luabind::functor<float> funct;
+    if (GEnv.ScriptEngine->functor("visual_memory_manager.get_visible_value", funct))
```

Comment:
`visual_memory_manager` module / global object is missing in default gamedata object. For me it caused non-stop exceptions / error logs / drops in performance

Reply:
Fixed in 947749cdee95f1dc9b5f251487dcd6929424f161.